### ISSUE TITLE
Accessibility audit -- remaining components

### ISSUE THREAD

**Issue by blackbaud-danbowman** [blackbaud/skyux1#753]
- Action bar
  - [x] Automation
  - [x] Keyboard
- Alert
  - [x] Automation
  - [x] Keyboard
- Avatar
  - [x] Automation
  - [x] Keyboard
- Button
  - [x] Automation
  - [x]  Keyboard
- Card
  - [x] Automation
  - [ ] Keyboard
- Carousel
  - [x] Automation
  - [ ] Keyboard (eventually could be improved to follow general keyboard pattern recommendations)
- Check
  - [x] Automation
  - [x] Keyboard
- Checklist 
  - [x] Automation
  - [ ] Keyboard
- Context Menu
  - [x] Automation
  - [x] Keyboard
- Datepicker (out for SKY UX 1.0)
  - [ ] Automation
  - [ ] Keyboard
- Date Range Picker  (out for SKY UX 1.0)
  - [ ] Automation
  - [ ] Keyboard
- Definition List
  - [x] Automation
  - [x] Keyboard
- Error 
  - [x] Automation
  - [x] Keyboard
- File Attachments
  - [x] Automation
  - [x] Keyboard
- Filter 
  - [x] Automation
  - [x] Keyboard
- Grids  (exclude table)
  - [ ] Automation
  - [ ] Keyboard
- Help button
  - [ ] Automation
  - [x] Keyboard
- Highlight
  - [x] Automation
  - [x] Keyboard
- Infinite scroll
  - [x] Automation
  - [x] Keyboard
- Key info
  - [x] Automation
  - [x] Keyboard
- Listbuilder
  - [ ] Automation
  - [ ] Keyboard
- Modal
  - [x] Automation
  - [x] Keyboard
- Navbar
  - [x] Automation
  - [x] Keyboard
- Page Summary
  - [x] Automation
  - [x] Keyboard
- Pagination
  - [x] Automation
  - [x] Keyboard
- Palette
  - [x] Automation
- Phone field
  - [x] Automation
  - [x] Keyboard
- Popover
  - [x] Automation
  - [x] Keyboard
- Reorder
  - [x] Automation
  - [ ] Keyboard
- Repeater
  - [x] Automation
  - [x] Keyboard
- Search
  - [x] Automation
  - [x] Keyboard
- Select field
  - [x] Automation
  - [ ] Keyboard
- Sort
  - [x] Automation
  - [x] Keyboard
- Tabset
  - [x] Automation
  - [x] Keyboard
- Text expand
  - [ ] Automation
  - [ ] Keyboard
- Tiles
  - [ ] Automation
  - [ ] Keyboard
- Toast
  - [x] Automation
  - [x] Keyboard
- Wait
  - [x] Automation
  - [x] Keyboard
- Wizard
  - [x] Automation
  - [x] Keyboard

**Comment by blackbaud-danbowman** [comment]
- [ ] Action bar

**Comment by Blackbaud-PatrickOFriel** [comment]
Hey @Blackbaud-MattGregg I went through the list and indicated each component that has accessibility automation included in its test suite.

**Comment by Blackbaud-MattGregg** [comment]
I believe all of the components are complete - automation, code review, manual keyboard testing.
